### PR TITLE
Vincent nmeregini ui ii

### PR DESCRIPTION
**Pull request by iamcristos**
Hello heres my PR

### CODE REVIEW

File: great-idea-website/css/index.css [hunk 1]
```diff
@@ -54,13 +54,155 @@ table {
...
+	border: 1px solid lavenderblush;
+	height: 50px;
+	display: inline-block;
+	background-color: lightgoldenrodyellow;
+	position: fixed;
+	z-index: +1;
+	width: 100%;
+	right: 5px;
+	top:0
+}
```
Comment (GianBarGian) [inline great-idea-website/css/index.css:97]: I love that you was able to give a fixed position to the nav bar. It looks like the position of the elements in it is a bit off. 
You could try to work more on `header div {` to move the logo to the right.

File: great-idea-website/index.html [hunk 2]
```diff
@@ -15,49 +15,71 @@
...
+        </div>
+        <!-- <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> -->
+      </nav>
+      <!-- <img class="logo" src="img/logo.png" alt="Great Idea! Company logo."> -->
+    </header>
+    <section id="top">
+      <div id="innovation">
+          <h1 >Innovation</h1>
+          <h2 class="in">On</h2>
+          <h1 >Demand</h1> <br>
```
Comment (GianBarGian) [inline great-idea-website/index.html:37]: Even if it's working fine I feel the way how you handled this `<h1>` is not the best. 
Try to check out the `<br>` HTML tag.

File: great-idea-website/index.html [hunk 3]
```diff
@@ -15,49 +15,71 @@
...
+      </div>
+      <div class="left" id="img-div">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>
+    <hr>
+    <section>
+      <div class="sec-section">
+          <div class="right">
+              <h1>Features</h1>
```
Comment (GianBarGian) [inline great-idea-website/index.html:48]: It's not a good practice to use more then one `<h1>` in the page. Normally it should work as the "title" of the page. 
You could substitute all of them with `<h2>` or `<h3>` .
You can read more about this here: https://www.searchenginejournal.com/header-tags-seo-best-practices/261835/

**Comment by GianBarGian** [comment]
Good job overall with your project. Apart minor issues, the project is well done.
The last thing I would like to add is to try to make your commits messages more meaningful, and explanatory on what is going on on your code. 
Well done Vincent!

**Comment by iamcristos** [comment]
Aiit thanks moving forward ill note and adjust to the changes

**Pull request merged**